cpp/benchmarks/ndsh/utilities.cpp

```diff
@@ -401,7 +401,7 @@ void generate_parquet_data_sources(double scale_factor,
   CUDF_BENCHMARK_RANGE();
 
   // Set the memory resource to the managed pool
-  auto old_mr = cudf::get_current_device_resource();
+  auto* old_mr = rmm::mr::get_current_device_resource();
```

Comment:
We need to use the RMM memory resource directly, since it’s required for constructing the RMM managed memory pool.